Update create-edit-or-delete-a-custom-user-view.md (#1505)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/create-edit-or-delete-a-custom-user-view.md

- **Unlicensed users** Select this box to find all the users who haven't been assigned a license. The results for this view can also include users who have an Exchange mailbox but don't have a license. To track those users specifically, use the filter **Unlicensed users with Exchange mailboxes or archives**. The results for this view can also include users who have an Exchange archive, but don't have a license.
    
- **Unlicensed users with Exchange mailboxes or archives** Select this box to show user accounts that were created in Exchange Online and have an Exchange mailbox, but weren't assigned an Office 365 license. The results of this filter include users who have or who were assigned an Exchange archive. 

> [!NOTE]

> > The unlicnesed users with exchange mailboxes works under the following situaions

Reviewer: ```suggestion
> The **Unlicensed users with Exchange mailboxes** filter works when:
```

> 1. The mailbox has been recently converted from shared to user and it has no license.

Reviewer: ```suggestion
1. The mailbox has been recently converted from **shared** to **user** and it has no license.
```

> 2. The mailbox has been recently migrated to O365 but license has not been assigned.

Reviewer: ```suggestion
2. The mailbox has been recently migrated to Office 365 but a license has not been assigned.
```

> 3. The mailbox has been created using powershell, and a license has not been assigned.

Reviewer: ```suggestion
3. The mailbox has been created using PowerShell, and a license has not been assigned.
```

> 4. A new mailbox is provisioned for user created on-premise with New-RemoteMailbox cmdlet.

Reviewer: ```suggestion
4. A new mailbox that has been created on-premise with a New-RemoteMailbox cmdlet is provisioned for the user.
```
